include/igraph_community.h

```diff
@@ -92,13 +92,13 @@ IGRAPH_EXPORT igraph_error_t igraph_community_walktrap(const igraph_t *graph,
                                             int steps,
                                             igraph_matrix_t *merges,
```

Comment:
This will have to become an `igraph_matrix_int_t` as well in the future, but it's not necessary to do it now.